components/list/list-item.js

```diff
@@ -122,6 +125,19 @@ class ListItem extends RtlMixin(LitElement) {
 			}
 		`;
 
+		const primaryAction = css`
+			:host([href]) .d2l-list-item-label {
+				width: 3.0rem;
+				z-index: 150;
+			}
+			:host([href]) {
+				--d2l-list-item-content-text-color: var(--d2l-color-celestine);
+			}
+			:host([href]) .d2l-list-item-link:hover + .d2l-list-item-content {
+				--d2l-list-item-content-text-decoration: underline;
```

Comment:
What about focus styles when they tab to the link with the keyboard?

Reply:
Thanks! I totally missed that.